modules/trusted-firmware-m/Kconfig.tfm.partitions

```
@@ -52,6 +56,7 @@ config TFM_PARTITION_INITIAL_ATTESTATION
 config TFM_PARTITION_PLATFORM
 	bool "Secure partition 'Platform'"
 	default y
+	depends on !TFM_SFN # Using PSA Framework version 1.0
```

```suggestion
	depends on !TFM_SFN # Currently using PSA Framework version 1.0
```

to indicate this is expected to be fixed soon